### DISCUSSION TITLE
Display current active branch name in tina sidebar

### DISCUSSION THREAD

**Discussion by Perni1984** [tinacms/tinacms#2504] [Ideas]
We are currently using the branch switcher experimental switch and also the TinaAdmin experimental switch. We really enjoy both experiences so far, one small thing we are missing is to immediately understand in which branch we are currently in, so we would propose to print the current selected branch like this besides "No changes / Unsaved Changes":

![image](https://user-images.githubusercontent.com/9039956/151311039-49fe9ded-61f7-47d3-a564-684f57ab3eba.png)

**Comment by logan-anderson** [comment]
@Perni1984 We totally agree! this has been on our minds too and I agree that is a good spot for the branch name to be. 

This is in our upcoming work and will be tracked here: #2263 

Thanks for bringing this up! your feedback is valuable 😄 

cc @Enigmatical
